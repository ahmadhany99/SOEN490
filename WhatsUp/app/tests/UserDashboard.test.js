import "react-native";
import React from "react";
import { NavigationContainer } from "@react-navigation/native";
import {
  render,
  waitFor,
  fireEvent,
  screen,
} from "@testing-library/react-native";
import UserDashboardScreen from "../screens/UserDashboard";

jest.useFakeTimers();

it("renders correctly", () => {
  const tree = render(
    <NavigationContainer>
      <UserDashboardScreen />
    </NavigationContainer>
  ).toJSON();
  expect(tree).toMatchSnapshot();
});

test("Clicking Location", async () => {
  render(
    <NavigationContainer>
      <UserDashboardScreen />
    </NavigationContainer>
  );
  await waitFor(() => {
    fireEvent.press(screen.getByText("Montreal, QC"));
  });
});

test("Clicking Notification", async () => {
  render(
    <NavigationContainer>
      <UserDashboardScreen />
    </NavigationContainer>
  );
  const temp = screen.queryAllByText("");
  await waitFor(() => {
    fireEvent.press(temp[0]);
  });
});

test("Clicking Filter", async () => {
  render(
    <NavigationContainer>
          <UserDashboardScreen />
    </NavigationContainer>
  );
  const temp = screen.queryAllByText("");
  await waitFor(() => {
    fireEvent.press(temp[1]);
  });
});

test("Clicking Event", async () => {
  render(
    <NavigationContainer>
      <UserDashboardScreen />
    </NavigationContainer>
  );
  const temp = screen.queryAllByText("");
  await waitFor(() => {
    fireEvent.press(screen.getByText("event"));
  });
});