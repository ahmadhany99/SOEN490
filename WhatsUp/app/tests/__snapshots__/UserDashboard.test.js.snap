// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<View
  accessibilityIgnoresInvertColors={true}
  style={
    Array [
      Object {
        "flex": 1,
        "paddingTop": 20,
      },
      Object {
        "backgroundColor": "#F5F5F5",
        "padding": 20,
      },
    ]
  }
>
  <Image
    source={
      Object {
        "uri": undefined,
      }
    }
    style={
      Array [
        Object {
          "bottom": 0,
          "left": 0,
          "position": "absolute",
          "right": 0,
          "top": 0,
        },
        Object {
          "height": undefined,
          "width": undefined,
        },
        undefined,
      ]
    }
  />
  <View
    style={
      Object {
        "left": "2.5%",
        "marginTop": "5%",
      }
    }
  >
    <View
      style={
        Object {
          "flexDirection": "row",
        }
      }
    >
      <View
        accessible={true}
        collapsable={false}
        focusable={false}
        onClick={[Function]}
        onResponderGrant={[Function]}
        onResponderMove={[Function]}
        onResponderRelease={[Function]}
        onResponderTerminate={[Function]}
        onResponderTerminationRequest={[Function]}
        onStartShouldSetResponder={[Function]}
        style={
          Object {
            "elevation": 2,
            "opacity": 1,
            "shadowOffset": Object {
              "height": 1,
              "width": 1,
            },
            "shadowOpacity": 0.2,
            "shadowRadius": 3,
          }
        }
      >
        <View
          style={
            Object {
              "flexDirection": "row",
            }
          }
        >
          <Text />
          <Text
            style={
              Object {
                "marginLeft": 2,
              }
            }
          >
            Montreal, QC
          </Text>
        </View>
      </View>
      <View
        accessible={true}
        collapsable={false}
        focusable={true}
        onClick={[Function]}
        onResponderGrant={[Function]}
        onResponderMove={[Function]}
        onResponderRelease={[Function]}
        onResponderTerminate={[Function]}
        onResponderTerminationRequest={[Function]}
        onStartShouldSetResponder={[Function]}
        style={
          Object {
            "elevation": 2,
            "opacity": 1,
            "position": "absolute",
            "right": 16,
            "shadowOffset": Object {
              "height": 1,
              "width": 1,
            },
            "shadowOpacity": 0.2,
            "shadowRadius": 3,
          }
        }
      >
        <View
          style={
            Object {
              "flexDirection": "row",
            }
          }
        >
          <Text />
          <Text
            style={
              Object {
                "marginLeft": 2,
              }
            }
          />
        </View>
      </View>
    </View>
    <Text
      style={
        Object {
          "color": "#969696",
          "fontSize": 12,
          "marginTop": "5%",
        }
      }
    >
      Today's January 26
    </Text>
    <Text
      style={
        Object {
          "color": "#100101",
          "fontSize": 25,
          "fontWeight": "bold",
          "marginBottom": 8,
        }
      }
    >
      Welcome, !
    </Text>
    <View
      style={
        Object {
          "flexDirection": "row",
        }
      }
    >
      <View
        accessible={true}
        collapsable={false}
        focusable={true}
        onClick={[Function]}
        onResponderGrant={[Function]}
        onResponderMove={[Function]}
        onResponderRelease={[Function]}
        onResponderTerminate={[Function]}
        onResponderTerminationRequest={[Function]}
        onStartShouldSetResponder={[Function]}
        style={
          Object {
            "elevation": 2,
            "flexDirection": "row",
            "marginTop": 5,
            "opacity": 1,
            "paddingHorizontal": 12,
            "shadowOffset": Object {
              "height": 1,
              "width": 1,
            },
            "shadowOpacity": 0.2,
            "shadowRadius": 3,
          }
        }
      >
        <View
          style={
            Object {
              "flexDirection": "row",
            }
          }
        >
          <Text />
          <Text
            style={
              Object {
                "marginLeft": 2,
              }
            }
          />
        </View>
      </View>
    </View>
    <Text
      style={
        Object {
          "color": "#100101",
          "fontSize": 16,
          "fontWeight": "bold",
          "marginTop": "4%",
        }
      }
    >
      Popular Events
    </Text>
    <View>
      <RCTScrollView
        data={
          Array [
            Object {
              "date": "May 21, 2022",
              "image": Object {
                "School": 1,
              },
              "key": 0,
              "organizer": "Concordia University",
              "title": "Orientation Week",
            },
            Object {
              "date": "May 21, 2022",
              "image": Object {
                "School": 1,
              },
              "key": 1,
              "organizer": "Concordia University",
              "title": "Orientation Week",
            },
            Object {
              "date": "May 21, 2022",
              "image": Object {
                "School": 1,
              },
              "key": 2,
              "organizer": "Concordia University",
              "title": "Orientation Week",
            },
            Object {
              "date": "May 21, 2022",
              "image": Object {
                "School": 1,
              },
              "key": 3,
              "organizer": "Concordia University",
              "title": "Orientation Week",
            },
            Object {
              "date": "May 21, 2022",
              "image": Object {
                "School": 1,
              },
              "key": 4,
              "organizer": "Concordia University",
              "title": "Orientation Week",
            },
            Object {
              "date": "May 21, 2022",
              "image": Object {
                "School": 1,
              },
              "key": 5,
              "organizer": "Concordia University",
              "title": "Orientation Week",
            },
          ]
        }
        getItem={[Function]}
        getItemCount={[Function]}
        keyExtractor={[Function]}
        onContentSizeChange={[Function]}
        onLayout={[Function]}
        onMomentumScrollBegin={[Function]}
        onMomentumScrollEnd={[Function]}
        onScroll={[Function]}
        onScrollBeginDrag={[Function]}
        onScrollEndDrag={[Function]}
        removeClippedSubviews={false}
        renderItem={[Function]}
        scrollEventThrottle={50}
        stickyHeaderIndices={Array []}
        style={Object {}}
        viewabilityConfigCallbackPairs={Array []}
      >
        <View>
          <View
            onLayout={[Function]}
            style={null}
          >
            <View
              accessible={true}
              collapsable={false}
              focusable={true}
              onClick={[Function]}
              onResponderGrant={[Function]}
              onResponderMove={[Function]}
              onResponderRelease={[Function]}
              onResponderTerminate={[Function]}
              onResponderTerminationRequest={[Function]}
              onStartShouldSetResponder={[Function]}
              style={
                Object {
                  "alignItems": "center",
                  "backgroundColor": "#FFFFFF",
                  "borderRadius": 19,
                  "borderWidth": 0,
                  "elevation": 4,
                  "flexDirection": "row",
                  "height": 81,
                  "justifyContent": "center",
                  "marginTop": 16,
                  "marginVertical": 5,
                  "opacity": 1,
                  "padding": 1,
                  "shadowColor": "#100101",
                  "shadowOffset": Object {
                    "height": 1,
                    "width": 1,
                  },
                  "shadowOpacity": 0.2,
                  "shadowRadius": 3,
                  "width": "95%",
                }
              }
            >
              <Image
                source={1}
                style={
                  Object {
                    "left": 0,
                    "position": "absolute",
                  }
                }
              />
              <View
                style={
                  Object {
                    "marginLeft": 30,
                  }
                }
              >
                <Text
                  style={
                    Object {
                      "color": "#100101",
                      "fontFamily": "Helvetica Neue",
                      "fontSize": 16,
                      "fontWeight": "bold",
                      "marginTop": "2%",
                    }
                  }
                >
                  Orientation Week
                </Text>
                <Text
                  style={
                    Object {
                      "color": "#969696",
                      "fontSize": 12,
                    }
                  }
                >
                  By 
                  Concordia University
                </Text>
                <View
                  style={
                    Object {
                      "flexDirection": "row",
                      "marginTop": 5,
                    }
                  }
                >
                  <Text />
                  <Text
                    style={
                      Object {
                        "color": "#100101",
                        "fontSize": 11,
                        "marginLeft": "2%",
                        "marginTop": "2%",
                      }
                    }
                  >
                    May 21, 2022
                  </Text>
                </View>
              </View>
              <View
                accessible={true}
                collapsable={false}
                focusable={true}
                onClick={[Function]}
                onResponderGrant={[Function]}
                onResponderMove={[Function]}
                onResponderRelease={[Function]}
                onResponderTerminate={[Function]}
                onResponderTerminationRequest={[Function]}
                onStartShouldSetResponder={[Function]}
                style={
                  Object {
                    "alignItems": "center",
                    "flexDirection": "row",
                    "height": 20,
                    "justifyContent": "center",
                    "marginLeft": 8,
                    "marginTop": 13,
                    "marginVertical": 5,
                    "opacity": 1,
                    "padding": 1,
                    "position": "absolute",
                    "right": 20,
                    "top": 5,
                    "width": 16,
                  }
                }
              >
                <Text />
              </View>
            </View>
          </View>
          <View
            onLayout={[Function]}
            style={null}
          >
            <View
              accessible={true}
              collapsable={false}
              focusable={true}
              onClick={[Function]}
              onResponderGrant={[Function]}
              onResponderMove={[Function]}
              onResponderRelease={[Function]}
              onResponderTerminate={[Function]}
              onResponderTerminationRequest={[Function]}
              onStartShouldSetResponder={[Function]}
              style={
                Object {
                  "alignItems": "center",
                  "backgroundColor": "#FFFFFF",
                  "borderRadius": 19,
                  "borderWidth": 0,
                  "elevation": 4,
                  "flexDirection": "row",
                  "height": 81,
                  "justifyContent": "center",
                  "marginTop": 16,
                  "marginVertical": 5,
                  "opacity": 1,
                  "padding": 1,
                  "shadowColor": "#100101",
                  "shadowOffset": Object {
                    "height": 1,
                    "width": 1,
                  },
                  "shadowOpacity": 0.2,
                  "shadowRadius": 3,
                  "width": "95%",
                }
              }
            >
              <Image
                source={1}
                style={
                  Object {
                    "left": 0,
                    "position": "absolute",
                  }
                }
              />
              <View
                style={
                  Object {
                    "marginLeft": 30,
                  }
                }
              >
                <Text
                  style={
                    Object {
                      "color": "#100101",
                      "fontFamily": "Helvetica Neue",
                      "fontSize": 16,
                      "fontWeight": "bold",
                      "marginTop": "2%",
                    }
                  }
                >
                  Orientation Week
                </Text>
                <Text
                  style={
                    Object {
                      "color": "#969696",
                      "fontSize": 12,
                    }
                  }
                >
                  By 
                  Concordia University
                </Text>
                <View
                  style={
                    Object {
                      "flexDirection": "row",
                      "marginTop": 5,
                    }
                  }
                >
                  <Text />
                  <Text
                    style={
                      Object {
                        "color": "#100101",
                        "fontSize": 11,
                        "marginLeft": "2%",
                        "marginTop": "2%",
                      }
                    }
                  >
                    May 21, 2022
                  </Text>
                </View>
              </View>
              <View
                accessible={true}
                collapsable={false}
                focusable={true}
                onClick={[Function]}
                onResponderGrant={[Function]}
                onResponderMove={[Function]}
                onResponderRelease={[Function]}
                onResponderTerminate={[Function]}
                onResponderTerminationRequest={[Function]}
                onStartShouldSetResponder={[Function]}
                style={
                  Object {
                    "alignItems": "center",
                    "flexDirection": "row",
                    "height": 20,
                    "justifyContent": "center",
                    "marginLeft": 8,
                    "marginTop": 13,
                    "marginVertical": 5,
                    "opacity": 1,
                    "padding": 1,
                    "position": "absolute",
                    "right": 20,
                    "top": 5,
                    "width": 16,
                  }
                }
              >
                <Text />
              </View>
            </View>
          </View>
          <View
            onLayout={[Function]}
            style={null}
          >
            <View
              accessible={true}
              collapsable={false}
              focusable={true}
              onClick={[Function]}
              onResponderGrant={[Function]}
              onResponderMove={[Function]}
              onResponderRelease={[Function]}
              onResponderTerminate={[Function]}
              onResponderTerminationRequest={[Function]}
              onStartShouldSetResponder={[Function]}
              style={
                Object {
                  "alignItems": "center",
                  "backgroundColor": "#FFFFFF",
                  "borderRadius": 19,
                  "borderWidth": 0,
                  "elevation": 4,
                  "flexDirection": "row",
                  "height": 81,
                  "justifyContent": "center",
                  "marginTop": 16,
                  "marginVertical": 5,
                  "opacity": 1,
                  "padding": 1,
                  "shadowColor": "#100101",
                  "shadowOffset": Object {
                    "height": 1,
                    "width": 1,
                  },
                  "shadowOpacity": 0.2,
                  "shadowRadius": 3,
                  "width": "95%",
                }
              }
            >
              <Image
                source={1}
                style={
                  Object {
                    "left": 0,
                    "position": "absolute",
                  }
                }
              />
              <View
                style={
                  Object {
                    "marginLeft": 30,
                  }
                }
              >
                <Text
                  style={
                    Object {
                      "color": "#100101",
                      "fontFamily": "Helvetica Neue",
                      "fontSize": 16,
                      "fontWeight": "bold",
                      "marginTop": "2%",
                    }
                  }
                >
                  Orientation Week
                </Text>
                <Text
                  style={
                    Object {
                      "color": "#969696",
                      "fontSize": 12,
                    }
                  }
                >
                  By 
                  Concordia University
                </Text>
                <View
                  style={
                    Object {
                      "flexDirection": "row",
                      "marginTop": 5,
                    }
                  }
                >
                  <Text />
                  <Text
                    style={
                      Object {
                        "color": "#100101",
                        "fontSize": 11,
                        "marginLeft": "2%",
                        "marginTop": "2%",
                      }
                    }
                  >
                    May 21, 2022
                  </Text>
                </View>
              </View>
              <View
                accessible={true}
                collapsable={false}
                focusable={true}
                onClick={[Function]}
                onResponderGrant={[Function]}
                onResponderMove={[Function]}
                onResponderRelease={[Function]}
                onResponderTerminate={[Function]}
                onResponderTerminationRequest={[Function]}
                onStartShouldSetResponder={[Function]}
                style={
                  Object {
                    "alignItems": "center",
                    "flexDirection": "row",
                    "height": 20,
                    "justifyContent": "center",
                    "marginLeft": 8,
                    "marginTop": 13,
                    "marginVertical": 5,
                    "opacity": 1,
                    "padding": 1,
                    "position": "absolute",
                    "right": 20,
                    "top": 5,
                    "width": 16,
                  }
                }
              >
                <Text />
              </View>
            </View>
          </View>
          <View
            onLayout={[Function]}
            style={null}
          >
            <View
              accessible={true}
              collapsable={false}
              focusable={true}
              onClick={[Function]}
              onResponderGrant={[Function]}
              onResponderMove={[Function]}
              onResponderRelease={[Function]}
              onResponderTerminate={[Function]}
              onResponderTerminationRequest={[Function]}
              onStartShouldSetResponder={[Function]}
              style={
                Object {
                  "alignItems": "center",
                  "backgroundColor": "#FFFFFF",
                  "borderRadius": 19,
                  "borderWidth": 0,
                  "elevation": 4,
                  "flexDirection": "row",
                  "height": 81,
                  "justifyContent": "center",
                  "marginTop": 16,
                  "marginVertical": 5,
                  "opacity": 1,
                  "padding": 1,
                  "shadowColor": "#100101",
                  "shadowOffset": Object {
                    "height": 1,
                    "width": 1,
                  },
                  "shadowOpacity": 0.2,
                  "shadowRadius": 3,
                  "width": "95%",
                }
              }
            >
              <Image
                source={1}
                style={
                  Object {
                    "left": 0,
                    "position": "absolute",
                  }
                }
              />
              <View
                style={
                  Object {
                    "marginLeft": 30,
                  }
                }
              >
                <Text
                  style={
                    Object {
                      "color": "#100101",
                      "fontFamily": "Helvetica Neue",
                      "fontSize": 16,
                      "fontWeight": "bold",
                      "marginTop": "2%",
                    }
                  }
                >
                  Orientation Week
                </Text>
                <Text
                  style={
                    Object {
                      "color": "#969696",
                      "fontSize": 12,
                    }
                  }
                >
                  By 
                  Concordia University
                </Text>
                <View
                  style={
                    Object {
                      "flexDirection": "row",
                      "marginTop": 5,
                    }
                  }
                >
                  <Text />
                  <Text
                    style={
                      Object {
                        "color": "#100101",
                        "fontSize": 11,
                        "marginLeft": "2%",
                        "marginTop": "2%",
                      }
                    }
                  >
                    May 21, 2022
                  </Text>
                </View>
              </View>
              <View
                accessible={true}
                collapsable={false}
                focusable={true}
                onClick={[Function]}
                onResponderGrant={[Function]}
                onResponderMove={[Function]}
                onResponderRelease={[Function]}
                onResponderTerminate={[Function]}
                onResponderTerminationRequest={[Function]}
                onStartShouldSetResponder={[Function]}
                style={
                  Object {
                    "alignItems": "center",
                    "flexDirection": "row",
                    "height": 20,
                    "justifyContent": "center",
                    "marginLeft": 8,
                    "marginTop": 13,
                    "marginVertical": 5,
                    "opacity": 1,
                    "padding": 1,
                    "position": "absolute",
                    "right": 20,
                    "top": 5,
                    "width": 16,
                  }
                }
              >
                <Text />
              </View>
            </View>
          </View>
          <View
            onLayout={[Function]}
            style={null}
          >
            <View
              accessible={true}
              collapsable={false}
              focusable={true}
              onClick={[Function]}
              onResponderGrant={[Function]}
              onResponderMove={[Function]}
              onResponderRelease={[Function]}
              onResponderTerminate={[Function]}
              onResponderTerminationRequest={[Function]}
              onStartShouldSetResponder={[Function]}
              style={
                Object {
                  "alignItems": "center",
                  "backgroundColor": "#FFFFFF",
                  "borderRadius": 19,
                  "borderWidth": 0,
                  "elevation": 4,
                  "flexDirection": "row",
                  "height": 81,
                  "justifyContent": "center",
                  "marginTop": 16,
                  "marginVertical": 5,
                  "opacity": 1,
                  "padding": 1,
                  "shadowColor": "#100101",
                  "shadowOffset": Object {
                    "height": 1,
                    "width": 1,
                  },
                  "shadowOpacity": 0.2,
                  "shadowRadius": 3,
                  "width": "95%",
                }
              }
            >
              <Image
                source={1}
                style={
                  Object {
                    "left": 0,
                    "position": "absolute",
                  }
                }
              />
              <View
                style={
                  Object {
                    "marginLeft": 30,
                  }
                }
              >
                <Text
                  style={
                    Object {
                      "color": "#100101",
                      "fontFamily": "Helvetica Neue",
                      "fontSize": 16,
                      "fontWeight": "bold",
                      "marginTop": "2%",
                    }
                  }
                >
                  Orientation Week
                </Text>
                <Text
                  style={
                    Object {
                      "color": "#969696",
                      "fontSize": 12,
                    }
                  }
                >
                  By 
                  Concordia University
                </Text>
                <View
                  style={
                    Object {
                      "flexDirection": "row",
                      "marginTop": 5,
                    }
                  }
                >
                  <Text />
                  <Text
                    style={
                      Object {
                        "color": "#100101",
                        "fontSize": 11,
                        "marginLeft": "2%",
                        "marginTop": "2%",
                      }
                    }
                  >
                    May 21, 2022
                  </Text>
                </View>
              </View>
              <View
                accessible={true}
                collapsable={false}
                focusable={true}
                onClick={[Function]}
                onResponderGrant={[Function]}
                onResponderMove={[Function]}
                onResponderRelease={[Function]}
                onResponderTerminate={[Function]}
                onResponderTerminationRequest={[Function]}
                onStartShouldSetResponder={[Function]}
                style={
                  Object {
                    "alignItems": "center",
                    "flexDirection": "row",
                    "height": 20,
                    "justifyContent": "center",
                    "marginLeft": 8,
                    "marginTop": 13,
                    "marginVertical": 5,
                    "opacity": 1,
                    "padding": 1,
                    "position": "absolute",
                    "right": 20,
                    "top": 5,
                    "width": 16,
                  }
                }
              >
                <Text />
              </View>
            </View>
          </View>
          <View
            onLayout={[Function]}
            style={null}
          >
            <View
              accessible={true}
              collapsable={false}
              focusable={true}
              onClick={[Function]}
              onResponderGrant={[Function]}
              onResponderMove={[Function]}
              onResponderRelease={[Function]}
              onResponderTerminate={[Function]}
              onResponderTerminationRequest={[Function]}
              onStartShouldSetResponder={[Function]}
              style={
                Object {
                  "alignItems": "center",
                  "backgroundColor": "#FFFFFF",
                  "borderRadius": 19,
                  "borderWidth": 0,
                  "elevation": 4,
                  "flexDirection": "row",
                  "height": 81,
                  "justifyContent": "center",
                  "marginTop": 16,
                  "marginVertical": 5,
                  "opacity": 1,
                  "padding": 1,
                  "shadowColor": "#100101",
                  "shadowOffset": Object {
                    "height": 1,
                    "width": 1,
                  },
                  "shadowOpacity": 0.2,
                  "shadowRadius": 3,
                  "width": "95%",
                }
              }
            >
              <Image
                source={1}
                style={
                  Object {
                    "left": 0,
                    "position": "absolute",
                  }
                }
              />
              <View
                style={
                  Object {
                    "marginLeft": 30,
                  }
                }
              >
                <Text
                  style={
                    Object {
                      "color": "#100101",
                      "fontFamily": "Helvetica Neue",
                      "fontSize": 16,
                      "fontWeight": "bold",
                      "marginTop": "2%",
                    }
                  }
                >
                  Orientation Week
                </Text>
                <Text
                  style={
                    Object {
                      "color": "#969696",
                      "fontSize": 12,
                    }
                  }
                >
                  By 
                  Concordia University
                </Text>
                <View
                  style={
                    Object {
                      "flexDirection": "row",
                      "marginTop": 5,
                    }
                  }
                >
                  <Text />
                  <Text
                    style={
                      Object {
                        "color": "#100101",
                        "fontSize": 11,
                        "marginLeft": "2%",
                        "marginTop": "2%",
                      }
                    }
                  >
                    May 21, 2022
                  </Text>
                </View>
              </View>
              <View
                accessible={true}
                collapsable={false}
                focusable={true}
                onClick={[Function]}
                onResponderGrant={[Function]}
                onResponderMove={[Function]}
                onResponderRelease={[Function]}
                onResponderTerminate={[Function]}
                onResponderTerminationRequest={[Function]}
                onStartShouldSetResponder={[Function]}
                style={
                  Object {
                    "alignItems": "center",
                    "flexDirection": "row",
                    "height": 20,
                    "justifyContent": "center",
                    "marginLeft": 8,
                    "marginTop": 13,
                    "marginVertical": 5,
                    "opacity": 1,
                    "padding": 1,
                    "position": "absolute",
                    "right": 20,
                    "top": 5,
                    "width": 16,
                  }
                }
              >
                <Text />
              </View>
            </View>
          </View>
        </View>
      </RCTScrollView>
      <RCTScrollView
        data={Array []}
        getItem={[Function]}
        getItemCount={[Function]}
        keyExtractor={[Function]}
        onContentSizeChange={[Function]}
        onLayout={[Function]}
        onMomentumScrollBegin={[Function]}
        onMomentumScrollEnd={[Function]}
        onScroll={[Function]}
        onScrollBeginDrag={[Function]}
        onScrollEndDrag={[Function]}
        removeClippedSubviews={false}
        renderItem={[Function]}
        scrollEventThrottle={50}
        stickyHeaderIndices={Array []}
        viewabilityConfigCallbackPairs={Array []}
      >
        <View />
      </RCTScrollView>
    </View>
  </View>
</View>
`;
